crates/ruff_python_formatter/resources/test/fixtures/ruff/expression/binary_expression.py

```diff
@@ -0,0 +1,27 @@
+(aaaaaaaa
+    + # trailing operator comment
+    b # trailing right comment
+)
+
+
+(aaaaaaaa # trailing left comment
+    +  # trailing operator comment
+    # leading right comment
+    b
+)
+
+
+# Black breaks the right side first for the following expressions:
```

Comment:
These are not yet handled by this PR (probably requires knowing what the parent is)